static/app/components/tooltip.tsx

```diff
@@ -466,4 +389,22 @@ const TooltipArrow = styled('span')`
   }
 `;
 
-export default Tooltip;
+interface AcceptanceTestTooltipProxyProps extends TooltipProps {
+  /**
+   * Stops tooltip from being opened during tooltip visual acceptance.
+   * Should be set to true if tooltip contains unisolated data (eg. dates)
+   */
+  disableForVisualTest?: boolean;
+}
+function AcceptanceTestTooltipProxy({
+  disableForVisualTest,
+  ...props
+}: AcceptanceTestTooltipProxyProps) {
+  if (IS_ACCEPTANCE_TEST) {
+    return disableForVisualTest ? null : <AcceptanceTestTooltip {...props} />;
```

Comment:
Does this work? Won't this mean that whatever the tooltip is wrapping just won't be rendered?

Reply:
Yes, correct. I need to forward pass props.children here